src/main/kotlin/racingcar/model/Car.kt

```diff
@@ -0,0 +1,37 @@
+package racingcar.model
+
+import camp.nextstep.edu.missionutils.Randoms
+
+class Car(name: String) {
```

Comment:
클래스 매개변수의 name과 필드의 name이 혼동되는것 같습니다. 

이를 구분할 수 있게 하거나 아니면 아래와 같이 선언해서 굳이 필드에 name에 할당하지 않고 사용할 수 있게 하면, 필드에 name을 두고 할당하는 과정이 없어도 되지 않을까 생각해 봅니다.

```suggestion
class Car(private val name: String) {
```